### ISSUE TITLE
The picture is full of noise use sdxl-turbo

### ISSUE THREAD

**Issue by zhaoyun0071** [huggingface/diffusers#6112]
### Describe the bug

from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_single_file(
    r"sd_xl_turbo_1.0_fp16.safetensors", torch_dtype=torch.float16, variant="fp16")
pipeline = pipeline.to("cuda")

prompt = "A cinematic shot of a baby racoon wearing an intricate italian priest robe."

image = pipeline(prompt=prompt, guidance_scale=0.0, num_inference_steps=1,height=512,width=512).images[0]
image.save('test.png')

![test](https://github.com/huggingface/diffusers/assets/35762050/f283b7c8-32b5-4bb8-a9d1-630f191d0366)

### Reproduction

from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_single_file(
    r"sd_xl_turbo_1.0_fp16.safetensors", torch_dtype=torch.float16, variant="fp16")
pipeline = pipeline.to("cuda")

prompt = "A cinematic shot of a baby racoon wearing an intricate italian priest robe."

image = pipeline(prompt=prompt, guidance_scale=0.0, num_inference_steps=1,height=512,width=512).images[0]
image.save('test.png')

### System Info

diffusers 0.24.0

**Comment by standardAI** [comment]
Hi @zhaoyun0071. You need to adjust the proper scheduler like this:
```py
from diffusers import EulerAncestralDiscreteScheduler
from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_single_file("sd_xl_turbo_1.0_fp16.safetensors",
                                                      torch_dtype=torch.float16, variant="fp16")
pipeline = pipeline.to("cuda")

prompt = "A cinematic shot of a baby racoon wearing an intricate italian priest robe."

scheduler_dict = {
                "beta_end": 0.012,
                "beta_schedule": "scaled_linear",
                "beta_start": 0.00085,
                "clip_sample": False,
                "interpolation_type": "linear",
                "num_train_timesteps": 1000,
                "prediction_type": "epsilon",
                "sample_max_value": 1.0,
                "set_alpha_to_one": False,
                "skip_prk_steps": True,
                "steps_offset": 1,
                "timestep_spacing": "trailing",
                "trained_betas": None,
            }
pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(scheduler_dict)
image = pipeline(prompt=prompt, guidance_scale=0.0, num_inference_steps=1, height=512, width=512).images[0]
image.save('test.png')
```

**Comment by zhaoyun0071** [comment]
> Hi @zhaoyun0071. You need to adjust the proper scheduler with parameters for SDXL-Turbo like this:
> 
> ```python
> from diffusers import EulerAncestralDiscreteScheduler
> from diffusers import StableDiffusionXLPipeline
> import torch
> 
> pipeline = StableDiffusionXLPipeline.from_single_file("sd_xl_turbo_1.0_fp16.safetensors",
>                                                       torch_dtype=torch.float16, variant="fp16")
> pipeline = pipeline.to("cuda")
> 
> prompt = "A cinematic shot of a baby racoon wearing an intricate italian priest robe."
> 
> scheduler_dict = {
>                 "beta_end": 0.012,
>                 "beta_schedule": "scaled_linear",
>                 "beta_start": 0.00085,
>                 "clip_sample": False,
>                 "interpolation_type": "linear",
>                 "num_train_timesteps": 1000,
>                 "prediction_type": "epsilon",
>                 "sample_max_value": 1.0,
>                 "set_alpha_to_one": False,
>                 "skip_prk_steps": True,
>                 "steps_offset": 1,
>                 "timestep_spacing": "trailing",
>                 "trained_betas": None,
>             }
> pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(scheduler_dict)
> image = pipeline(prompt=prompt, guidance_scale=0.0, num_inference_steps=1, height=512, width=512).images[0]
> image.save('test.png')
> ```

thanks

**Comment by bghira** [comment]
or just avoid from_single_file... it's not meant for atypical model types

**Comment by standardAI** [comment]
Can I open a PR to solve this problem after the current `from_single_file` refactoring?
@DN6

**Comment by standardAI** [comment]
This problem is still continuing. Do you think it is necessary to open a PR?
@sayakpaul @DN6

**Comment by sayakpaul** [comment]
I think we make it clear from our Turbo guide. Doesn’t that help?

cc; @yiyixuxu

**Comment by standardAI** [comment]
> I think we make it clear from our Turbo guide. Doesn’t that help?
> 
> cc; @yiyixuxu

I examined the guide. It emphasizes using `timestep_spacing='trailing'`; but `StableDiffusionXLPipeline.from_single_file` doesn't work as expected out-of-the-box, it needs to adjust its scheduler.

**Comment by sayakpaul** [comment]
We can add that information in the guide maybe? Feel free to open a PR. Cc: @DN6
